ci/core/pipeline.yml

```diff
@@ -113,8 +113,12 @@ jobs:
   -  #@ buck_test(component)
   -  #@ component_bats_tests(component)
   -  #@ build_edge_image(component)
+  #@ if component == "api-keys":
```

Comment:
this is not a good pattern.
Never put a conditional inside a loop to identify which iteration of the loop you are on... what's the point of having the loop in that case?